libs/core/src/components.d.ts

```diff
@@ -597,7 +663,7 @@ declare namespace LocalJSX {
          */
         "invalid"?: boolean;
         /**
-          * Text to be displayed as the form label
+          * Text to be displayed as the input label
```

Comment:
Should we say textarea instead of input, so it matches other descriptions? This is a nitpick and not a blocker.